docs/stores/eks/eks.md

```diff
@@ -0,0 +1,51 @@
+# EKS store
+
+Kubeswitch can discover EKS clusters from AWS.
+
+At the moment, Kubeswitch works only with [named profiles](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html), make sure it is working before setting up the store.
+A good test is to follow the "(Create kubeconfig automatically)[https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html#create-kubeconfig-automatically]" documentation as Kubeswitch will output the same configuration (with some exceptions pending implementation).
+
+## Setup
+
+Please make sure the `aws` cli is intalled and on your `PATH` in a version providing the [`aws eks get-token`](https://docs.aws.amazon.com/cli/latest/reference/eks/get-token.html) command.
```

Comment:
```suggestion
Please make sure the `aws` cli is intalled and on your `PATH` in a version providing the [`aws eks get-token`](https://docs.aws.amazon.com/cli/latest/reference/eks/get-token.html) command (requires version 1.16.156 or later of the AWS CLI).
This is because the generated kubeconfigs use the AWS CLI as a credential helper.
```